### PR TITLE
[Storage] Updates to the outdated client packages part for storage service

### PR DESCRIPTION
**Pull request by HarshaNalluru**
## Issue
The overview page - Following docs still refer to `azure-storage` package.
> https://review.docs.microsoft.com/en-us/javascript/api/overview/azure/storage?view=azure-node-preview&branch=master

Issue - https://github.com/Azure/azure-sdk-for-js/issues/4728

## Description
Updated the markdown with the new V12 packages and removed the older code snippets.
After the updates, the markdown similar to the following screenshot!

Fixes https://github.com/Azure/azure-sdk-for-js/issues/4728

### CODE REVIEW

File: docs-ref-services/storage.md [hunk 1]
```diff
@@ -14,49 +14,26 @@ ms.service: storage
...
+|Service| NPM link| Examples|Source Code|Getting Started Guide|
+|---|---|---|--|--|
+|**Storage Blob**|[@azure/storage-blob](https://www.npmjs.com/package/@azure/storage-blob)|[storage-blob-typescript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-blob-typescript/)<br> [storage-blob-javascript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-blob-javascript/)|[Source Code - Blob](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob)|Read and write objects and files from [Azure Storage Blob](https://docs.microsoft.com/azure/storage/storage-nodejs-how-to-use-blob-storage)|
+|**Storage File Share**|[@azure/storage-file-share](https://www.npmjs.com/package/@azure/storage-file-share)|[storage-file-share-typescript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-file-share-typescript/)<br> [storage-file-share-javascript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-file-share-javascript/)|[Source Code - File Share](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-file-share)|#ToBeAdded|
```
Comment (kaerm) [inline docs-ref-services/storage.md]: instead of to be added i'd point to our readme in this case

File: docs-ref-services/storage.md [hunk 2]
```diff
@@ -14,49 +14,26 @@ ms.service: storage
 
 # Azure Storage modules for Node.js
```
Comment (kaerm) [inline docs-ref-services/storage.md]: For all docs.microsoft.com links remove /en-us from the link

File: docs-ref-services/storage.md [hunk 3]
```diff
@@ -14,49 +14,26 @@ ms.service: storage
...
-    (error, result) => {
-      if (error) return console.log(error);
-      console.dir(result, { depth: null, colors: true });
-    }
-  );
-});
-```
+Read more about the client packages here - [Azure Storage Client Libraries For JavaScript](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/)
 
 ## Management Package
```
Comment (kaerm) [inline docs-ref-services/storage.md]: Delete the bottom section with samples.. These links are broken

File: docs-ref-services/storage.md [hunk 4]
```diff
@@ -14,49 +14,26 @@ ms.service: storage
 
 # Azure Storage modules for Node.js
 
-Use the Azure Storage client module to:
+Azure Storage is a Microsoft-managed service providing cloud storage that is highly available, secure, durable, scalable, and redundant. The following libraries in JavaScript make it easy to consume Microsoft Azure Storage service.
```
Comment (kaerm) [inline docs-ref-services/storage.md]: Add link to https://docs.microsoft.com/azure/storage/

File: docs-ref-services/storage.md [hunk 5]
```diff
@@ -14,49 +14,26 @@ ms.service: storage
 
 # Azure Storage modules for Node.js
```
Comment (kaerm) [inline docs-ref-services/storage.md]: are libraries called modules in JS? Also if they work for everything just write JavaScript

File: docs-ref-services/storage.md [hunk 6]
```diff
@@ -14,49 +14,26 @@ ms.service: storage
...
+|**Storage File Share**|[@azure/storage-file-share](https://www.npmjs.com/package/@azure/storage-file-share)|[storage-file-share-typescript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-file-share-typescript/)<br> [storage-file-share-javascript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-file-share-javascript/)|[Source Code - File Share](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-file-share)|#ToBeAdded|
+|**Storage File Datalake**|[@azure/storage-file-datalake](https://www.npmjs.com/package/@azure/storage-file-datalake)|[storage-file-datalake-typescript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-file-datalake-typescript/)<br> [storage-file-datalake-javascript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-file-datalake-javascript/)|[Source Code - File Datalake](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-file-datalake)|#ToBeAdded|
```
Comment (kaerm) [inline docs-ref-services/storage.md]: same here - point to our readme

File: docs-ref-services/storage.md [hunk 7]
```diff
@@ -14,49 +14,26 @@ ms.service: storage
...
+|**Storage Queue**|[@azure/storage-queue](https://www.npmjs.com/package/@azure/storage-queue)|[storage-queue-typescript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-queue-typescript/)<br> [storage-queue-javascript-examples](https://docs.microsoft.com/en-us/samples/azure/azure-sdk-for-js/storage-queue-javascript/)|[Source Code - Queue](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-queue)|Send and receive messages between cloud-connected applications with <br>#ToBeAdded|
```
Comment (kaerm) [inline docs-ref-services/storage.md]: Queue quickstart exists https://docs.microsoft.com/azure/storage/queues/storage-quickstart-queues-nodejs

File: docs-ref-services/storage.md [hunk 8]
```diff
@@ -14,49 +14,26 @@ ms.service: storage
...
+|**Storage Table**|[azure-storage](https://www.npmjs.com/package/azure-storage)<br>(Legacy)|#ToBeAdded|#ToBeAdded|Read and write large structured data with [Azure Table storage](https://docs.microsoft.com/azure/storage/storage-nodejs-how-to-use-table-storage)|
```
Comment (kaerm) [inline docs-ref-services/storage.md]: I'd just leave the table entries empty

File: docs-ref-services/storage.md (hunk 4 above)
Comment (kaerm) [inline docs-ref-services/storage.md]: Also, what's wrong with the old description? :)

Comment (kaerm) [inline docs-ref-services/storage.md]: We no longer use Microsoft Azure, just Azure

File: docs-ref-services/storage.md (hunk 3 above)
Comment (HarshaNalluru) [reply docs-ref-services/storage.md]: Kept this part and removed the rest.

> ## Samples

> Explore more [sample Node.js code](https://azure.microsoft.com/resources/samples/?platform=nodejs) you can use in your apps.

File: docs-ref-services/storage.md (hunk 7 above)
Comment (HarshaNalluru) [reply docs-ref-services/storage.md]: Thanks

File: docs-ref-services/storage.md (hunk 8 above)
Comment (HarshaNalluru) [reply docs-ref-services/storage.md]: Just blank cells?

File: docs-ref-services/storage.md (hunk 4 above)
Comment (HarshaNalluru) [reply docs-ref-services/storage.md]: Hmmm.. let me clarify.
- Moved the older bullet list of descriptions to the table below.
- Copied this new description from our readme at azure-sdk-for-js repo.

**Comment by HarshaNalluru** [comment]
@kaerm, I think I've addressed all your feedback.
Please re-review and let me know if you find anything else.

**Comment by HarshaNalluru** [comment]
> seems like the table isn't generating properly now?

VS Code was smart enough to understand improper syntax.. fixed now.

**Comment by jeremymeng** [comment]
need to address the warning by removing `en-us` from samples links. Also need to update/remove the example?

**Comment by HarshaNalluru** [comment]
> need to address the warning by removing en-us from samples links

Removed.. warning report was from an older commit

> Also need to update/remove the example?

which?

**Comment by jeremymeng** [comment]
> which

ah nvm, it's a management plane example.

File: docs-ref-services/storage.md [hunk 9]
```diff
@@ -1,62 +1,41 @@
...
-ms.devlang: nodejs
+ms.devlang: javascript
 ms.service: storage
 ---
 
-# Azure Storage modules for Node.js
+# Azure Storage libraries for JavaScript
 
-Use the Azure Storage client module to:
+[Azure Storage](https://docs.microsoft.com/azure/storage/) is a Microsoft-managed service providing cloud storage that is highly available, secure, durable, scalable, and redundant. The following libraries in JavaScript make it easy to consume Azure Storage service.
```
Comment (ramya-rao-a) [inline docs-ref-services/storage.md]: Why the hyphenated `Microsoft-manager`?

File: docs-ref-services/storage.md [hunk 10]
```diff
@@ -1,62 +1,41 @@
...
-    (error, result) => {
-      if (error) return console.log(error);
-      console.dir(result, { depth: null, colors: true });
-    }
-  );
-});
-```
+Find more getting started guides at [Browse code samples](https://azure.microsoft.com/resources/samples/)
 
 ## Management Package
```
Comment (ramya-rao-a) [inline docs-ref-services/storage.md]: Can we have a 1 line description that tells the user what a "management package" is?

File: docs-ref-services/storage.md (hunk 9 above)
Comment (HarshaNalluru) [reply docs-ref-services/storage.md]: Not me. Copy/pasted from the `azure-sdk-for-js`/`storage` readme.

File: docs-ref-services/storage.md (hunk 10 above)
Comment (HarshaNalluru) [reply docs-ref-services/storage.md]: That's a very interesting question.

Which one do you like?
> Management packages enable you to create, update, delete, and list resources in your Azure subscription.

or

> Azure storage management package enables you to create, update, delete, and list storage resources in your Azure subscription.

Or anything else you'd suggest?

File: docs-ref-services/storage.md (hunk 9 above)
Comment (HarshaNalluru) [inline docs-ref-services/storage.md]:
```suggestion
[Azure Storage](https://docs.microsoft.com/azure/storage/) is a Microsoft managed service providing cloud storage that is highly available, secure, durable, scalable, and redundant. The following libraries in JavaScript make it easy to consume Azure Storage service.
```

**Comment by kaerm** [comment]
@HarshaNalluru are we done with edits here? I can merge

**Comment by HarshaNalluru** [comment]
> @HarshaNalluru are we done with edits here? I can merge

Please merge, @kaerm !

**Pull request merged**